tests/tests/tier0/bluechi-change-port-with-c-cmd-option/test_bluechi_change_port_with_c_cmd_option.py

```diff
@@ -29,9 +33,11 @@ def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
     assert ctrl.wait_for_unit_state_to_be(bluechi_controller_str, "active")
 
     # Check if port 8421 is listening and 8420 is disconnected
-    _, output = ctrl.exec_run("lsof -i:8421")
-    assert "bluechi-c" in str(output)
-    _, output = ctrl.exec_run("lsof -i:8420")
+    res, output = ctrl.exec_run("bash /var/cmd-on-port.sh 8421")
+    assert res == 0
```

Comment:
This currently fails due to 
```
AssertionError: assert 127 == 0
```
Maybe adding `exit 0` at the end of the script solves this?